csharp/src/Ice/OutputStream.cs

```diff
@@ -18,7 +18,7 @@ public interface IStreamableStruct // value with the value-type semantics
         public void IceWrite(OutputStream ostr);
     }
 
-    public delegate void OutputStreamWriter<in T>(OutputStream os, T value);
+    public delegate void OutputStreamWriter<T>(OutputStream os, T value);
```

Comment:
Do you know why this `in` is no longer needed?

Reply:
I will revert that, this is required for helper dictionaries to always use IReadOnlyDictionary for writing. Shouldn't have change that, at least not in this PR